### PR TITLE
Acceptance: Harvest Borrow

### PR DESCRIPTION
**Pull request by denalimarsh**
Acceptance instructions can be found in `acceptance/1_harvest_borrow/1_harvest_borrow.md`.

### CODE REVIEW

File: acceptance/1_harvest_borrow/1_harvest_borrow.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+First, recover the oracle account and set up the pricefeed state.
+
+```bash
+# Recover oracle account
+docker exec -it generated_kavanode_1 kvcli keys add oracle --recover
+# Enter mnemonic
+desert october mammal tuition illness album engine solid enjoy harvest symptom rely camera unable okay avocado actual oppose remember lady dove canal argue cave
+
+# Set asset prices in price feed
+docker exec -it generated_kavanode_1 kvcli tx pricefeed postprice usdx:usd 1 99999999999 --from oracle
```
Comment (karzak) [inline acceptance/1_harvest_borrow/1_harvest_borrow.md:44]: Set prices in the genesis file.

File: acceptance/1_harvest_borrow/1_harvest_borrow.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+```bash
+# Recover user account
+docker exec -it generated_kavanode_1 kvcli keys add user --recover
+# Enter mnemonic
+arrive guide way exit polar print kitchen hair series custom siege afraid shrug crew fashion mind script divorce pattern trust project regular robust safe
+
+# Load USDX onto testing account by creating a CDP
+# NOTE: the given user account is already loaded with USDX, but this command enables feature testing
+#       on different accounts which have not already been pre-loaded.
+docker exec -it generated_kavanode_1 kvcli tx cdp create 1000000000bnb 100000000usdx bnb-a --from user --gas 500000
```
Comment (karzak) [inline acceptance/1_harvest_borrow/1_harvest_borrow.md:61]: Use an account pre-funded with USDX.

**Review by karzak — CHANGES_REQUESTED** [review]
Instructions look fine. Update genesis file to avoid the need to 

1. Post prices manually
2. Draw usdx using a CDP

File: acceptance/1_harvest_borrow/1_harvest_borrow.md (hunk 2 above)
Comment (denalimarsh) [reply acceptance/1_harvest_borrow/1_harvest_borrow.md]: Removed this section in https://github.com/Kava-Labs/kvtool/pull/17/commits/213ecb786d7cf968cbffd99b8a7877e37f7561ed (account already has usdx available)

File: acceptance/1_harvest_borrow/1_harvest_borrow.md (hunk 1 above)
Comment (denalimarsh) [reply acceptance/1_harvest_borrow/1_harvest_borrow.md]: Updated in https://github.com/Kava-Labs/kvtool/pull/17/commits/213ecb786d7cf968cbffd99b8a7877e37f7561ed

**Review by karzak — APPROVED** [review]

**Pull request merged**